apps/system/js/browser_context_menu.js

```diff
@@ -171,6 +193,31 @@
       }));
     },
 
+    pinUrl: function(url) {
+      var data = {
+        type: 'url',
+        url: url,
+        iconable: false,
+        title: this.app.title,
+        theme: this.app.themeColor
+      };
+
+      this.app.getScreenshot(function() {
+        data.screenshot = this.app._screenshotBlob;
+        this.app.getSiteIconUrl()
```

Comment:
I would leave it inconsistent and file a bug for the incorrect one, but is only my opinion.